README.md

```diff
@@ -28,16 +29,27 @@ like the Clang-as-a-library compiler implementation.
 
 libVersioningCompiler uses the cmake build system.
 If you want to install libVersioningCompiler
-
+install the dependencies with LLVM version `${LLVM_V}`:
+
+```bash
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+[sudo] ./llvm.sh ${LLVM_V} all
+[sudo] apt-get install -y git cmake uuid-dev g++-10 libz-dev
+[optional][sudo] update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-${LLVM_V} 100
+[optional][sudo] update-alternatives --install /usr/bin/clang clang /usr/bin/clang-${LLVM_V} 100
+[optional][sudo] update-alternatives --install /usr/bin/opt opt /usr/bin/opt-${LLVM_V} 100
+```
```

Comment:
Need to add a section

```
OR manually specify your path to pre-installed LLVM version using ....

```

Reply:
I had updated the readme with how to specify a custom LLVM installation, let me know what you think about it